src/main/java/org/llorllale/cactoos/matchers/InputHasContent.java

```diff
@@ -72,22 +62,19 @@ public InputHasContent(final Text text) {
      * @param mtr Matcher of the text
      */
     public InputHasContent(final Matcher<String> mtr) {
-        super();
-        this.matcher = mtr;
-    }
-
-    @Override
-    public boolean matchesSafely(final Input item) {
-        return this.matcher.matches(
-            new UncheckedText(
-                new TextOf(item)
-            ).asString()
+        super(
+            // @checkstyle IndentationCheck (1 line)
```

Comment:
@borysfan You can use just on `// @checkstyle IndentationCheck (5 lines)` here.

Reply:
Done.

Reply:
Done